.ado/templates/react-native-init.yml

```diff
@@ -0,0 +1,80 @@
+#
+parameters:
+  language:
+
+steps:
+  - checkout: self # self represents the repo where the initial Pipelines YAML file was found
+    clean: true # whether to fetch clean each time
+    # fetchDepth: 2 # the depth of commits to ask Git to fetch
+    lfs: false # whether to download Git-LFS files
+    submodules: false # set to 'true' for a single level of submodules or 'recursive' to get submodules of submodules
+    persistCredentials: false # set to 'true' to leave the OAuth token in the Git config after the initial fetch
+
+  # First do a build of the local package, since we point the cli at the local files, it needs to be pre-built
+  - task: CmdLine@2
+    displayName: yarn install (local react-native-windows)
+    inputs:
+      script: yarn install --frozen-lockfile
+
+  - task: CmdLine@2
+    displayName: yarn build (local react-native-windows)
+    inputs:
+      script: yarn build
+      workingDirectory: vnext
+
+  # yarn ends up copying the whole node_modules folder when doing an install of a file package
+  # Delete node_modules, so that resolution is more like when installing from a published npm package
+  - task: CmdLine@2
+    displayName: Remove node_modules
+    inputs:
+      script: rd /S /Q node_modules
+      workingDirectory: vnext
+
+  - task: CmdLine@2
+    displayName: Install react-native cli
+    inputs:
+      script: npm install -g react-native-cli
+
+  - task: CmdLine@2
+    displayName: Init new project
+    inputs:
+      script: react-native init testcli --version 0.60.6
```

Comment:
Parameterize version.

Reply:
done

---
In reply to: [328923250](https://github.com/microsoft/react-native-windows/pull/3262#discussion_r328923250) [](ancestors = 328923250)